HTML/1. intro.md

```diff
@@ -0,0 +1,39 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language that is used to create webpages, so it's not like a programming language like Python
+___
+HTML is structured by tags and content which make up elements. There ways of declaring an element.
+
+1. The first way of declaring elements is with an opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `<` to begin and a `/>` to end. Everything else between the opening and closing tags is either content or other nested elements
```

Comment:
```suggestion
1. The first way of declaring elements is with an opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `</` to begin and a `>` to end. Everything else between the opening and closing tags is either content or other nested elements
```

`<html>...</html>`

Reply:
Eyyy I am succesfull at glitching github

I committed the suggestion but it actually isn't committed sry lul

You still have to fix this @MrIntellectualGuy

Reply:
Actually, I did commit it

how
wut
yay
les
go

@CrimsonCodes0 I committed your suggestion somehow